### DISCUSSION TITLE
Hyper links work on IDE's and Github.dev, but not Github

### DISCUSSION THREAD

**Discussion by Josh9182** [community/community#125212] [Programming Help]
### Select Topic Area

Product Feedback

### Body

Hey! I have been using Github for a couple months and use Visual Studio Code, Pycharm, and Github.dev to test code. I imported a Jupyter Notebook exploring data analytics, however the hyper links take me back to my repositories home page as opposed to the correct location inside the file. 

On everything, Jupyter Notebook, VSC, Pycharm, Github.dev, it all works! On github however, it doesn't I do not understand why. (Spaces were put to show the code.)

Code:

[How many posts were made on each platform?](# how-many-posts-were-made-on-each-platform)

Title:

## < a id='how-many-posts-were-made-on-each-platform'></a> How many posts were made on each platform?

It works on everything except github, why is this?

**Comment by RafaelJohn9** [comment]
Can you send the link  to your repo so that I can help?

**Reply by Josh9182** [reply]
Terribly sorry! I can create a whole new repo if you would like since the OG repo is just an ipynb file. Maybe you can try it yourself, it's bizarre. Using VSC and Jupyter lets you click through the page, the minute its put on GitHub it just shuts down.
